### ISSUE TITLE
newly added subscribers not immediately reflected in list

### ISSUE THREAD

**Issue by james-baker** [buttondown-email/roadmap#2027]
Be looking at most-recent-first date-ordered list of subscribers:
https://buttondown.email/subscribers?order=-date
Hit the blue "Add" button and finish that modal.
See a toast for "subscriber <email> added" (or something like that)

Expected: that just-added email should pop in at the top of the list, reflecting new current reality.
Actual: no response/change from the list already on the page

Hitting F5 does include the new subscriber in the fully-re-fetched list.
I don't think this depends on regular vs unactivated status. I've used "automatically activate"
And I'm not certain but, this feels like a regression of behavior that used to be there.

**Comment by jmduke** [comment]
@james-baker I suspect this is because we just automatically push newly added subscribers to the end of the list. Do you mind trying this on your end and seeing if they're at the bottom of the page?

**Comment by james-baker** [comment]
Yes, confirmed: new items do show up at the end/bottom of the list, disregarding sort order.

**Comment by jmduke** [comment]
Cool. This will be a _very_ annoying one to fix, but at least thankfully we have some clientside ordering in place ;)

**Comment by jmduke** [comment]
That is an entirely more sensible solution!

**Comment by james-baker** [comment]
Today's exposure: Same as last message - Wrote an email, sent it, came back to the emails page 10 minutes, scrolled to the bottom to see the thing-i-just-sent-that-should-be-at-the-top AND I noticed that it also has a stale status, "About to send" though it's already been sent (and "status": "sent" in the post-send analytics polling) ten minutes ago. So there's data staleness beyond the lack of a fresh sort.

**Comment by james-baker** [comment]
Hey did this just get fixed? I wrote a new email, published it, then clicked "Emails" a few minutes later, and "POP" it came in at the top of the list!

**Comment by jmduke** [comment]
uhhhhhhhhhhh

_no_?

😅 

I have been tweaking the internal state management logic and cleaning things up a good deal, but definitely did not _intentionally_ fix this. I suspect it might have been just the backing emails store refreshing naturally over time. I'll leave this open for a little longer.
